Formula/teleport.rb

```diff
@@ -0,0 +1,18 @@
+class Teleport < Formula
+  desc "Modern SSH server for teams managing distributed infrastructure"
+  homepage "https://gravitational.com/teleport"
+  url "https://github.com/gravitational/teleport/releases/download/v1.3.2/teleport-v1.3.2-darwin-amd64-bin.tar.gz"
+  sha256 "447fa78ace7d488a3b75ae834059a5dcaccb182f338bb97b8f9414e3a4947428"
+
+  def install
+    prefix.install Dir["*"]
+    bin.install_symlink prefix/"tctl"
+    bin.install_symlink prefix/"teleport"
+    bin.install_symlink prefix/"tsh"
+    share.install_symlink prefix/"app"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/teleport version")
```

Comment:
Any chance of a more detailed test here? We prefer not to test `version` or `help` output unless nothing else is available.